README.md

```diff
@@ -39,78 +39,96 @@
+- [Semgrep Code's Pro rules](https://semgrep.dev/products/semgrep-code): 600+ high confidence rules written by Semgrep's security research team
+- [Semgrep Code's Pro engine](https://semgrep.dev/products/pro-engine/): An advanced code analysis engine, designed to detect complex vulnerabilities, and reduce false positives
 
-3.  To run Semgrep Supply Chain, [contact the Semgrep team](https://semgrep.dev/contact-us).
-    Visit the [full documentation](https://semgrep.dev/docs/getting-started/) to learn more.
+3.  Go to your app's root directory and run `semgrep ci`. This will scan your project to check for vulnerabilities in your source code and its dependencies.
 
 ### Semgrep Ecosystem
 
 The Semgrep ecosystem includes the following products:
 
-- Semgrep OSS Engine - The open-source engine at the heart of everything (this project).
-- [Semgrep Cloud Platform (SCP)](https://semgrep.dev/login) - Deploy, manage, and monitor SAST and SCA at scale using Semgrep, with [free and paid tiers](https://semgrep.dev/pricing). Integrates with continuous integration (CI) providers such as GitHub, GitLab, CircleCI, and more.
-- [Semgrep Code](https://semgrep.dev/products/semgrep-code) - Scan your code with Semgrep's Pro rules and Semgrep Pro Engine to find OWASP Top 10 vulnerabilities and protect against critical security risks specific to your organization. Semgrep Code is available on both [free and paid tiers](https://semgrep.dev/pricing).
+- [Semgrep Code](https://semgrep.dev/products/semgrep-code) - Scan your code with Semgrep's proprietary rules (written by our Security Research team) using our cross-file and cross-function analysis. Designed to find OWASP Top 10 vulnerabilities and protect against critical security risks. Semgrep Code is available on both [free and paid tiers](https://semgrep.dev/pricing).
 - [Semgrep Supply Chain (SSC)](https://semgrep.dev/products/semgrep-supply-chain) - A high-signal dependency scanner that detects reachable vulnerabilities in open source third-party libraries and functions across the software development life cycle (SDLC). Semgrep Supply Chain is available on both [free and paid tiers](https://semgrep.dev/pricing).
+- [Semgrep Secrets)](https://semgrep.dev/blog/2023/introducing-semgrep-secrets/)[NEW!] - Secrets detection that uses semantic analysis, improved entropy analysis, and validation together to accurately detect sensitive credentials in developer workflows. Sign up for our [private beta](https://get.semgrep.dev/secrets-beta-request.html) to request early access to the product.
+- [Semgrep Cloud Platform (SCP)](https://semgrep.dev/login) - Deploy, manage, and monitor Semgrep at scale, with [free and paid tiers](https://semgrep.dev/pricing). Integrates with continuous integration (CI) providers such as GitHub, GitLab, CircleCI, and more.
+- Semgrep OSS Engine - The open-source engine and community-contributed rules at the heart of everything (this project).
```

Comment:
Could we link to https://semgrep.dev/docs/cli-reference/ for `Semgrep OSS Engine` for consistency?